bin/view_godocs

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# This is a convenience script to start 
+
+full_path=$(pwd)
+full_parent_path=${full_path%/*}
+parent_dir=${full_path##*/}
```

Comment:
Maybe a typo... The parent dir is the end part of full_parent_path not full_path
```suggestion
parent_dir=${full_parent_path##*/}
```